ivy_tests/test_ivy/test_functional/test_nn/test_losses.py

```diff
@@ -15,23 +15,17 @@
         max_value=1,
         allow_inf=False,
         min_num_dims=1,
-        max_num_dims=1,
-        min_dim_size=2,
     ),
     dtype_and_pred=helpers.dtype_and_values(
         available_dtypes=helpers.get_dtypes("float"),
         min_value=1e-04,
```

Comment:
Ideally, we should also not limit the min value, if it causes issues due to precision errors, we should use the safety factor.